pcsx2-qt/Settings/AdvancedSettingsWidget.cpp

```diff
@@ -96,8 +96,8 @@ AdvancedSettingsWidget::AdvancedSettingsWidget(SettingsWindow* dialog, QWidget*
 		   "end of the block, not on the instruction which caused the exception. Refer to the console to see the address where the invalid "
 		   "access occurred."));
 
-	dialog->registerWidgetHelp(m_ui.vu0RoundingMode, tr("VU0 Rounding Mode"), tr("Chop / Zero (Default)"), tr(""));
```

Comment:
This was done on purpose for visual spacing.

Reply:
I removed those spacings because it was unspaced in Big Picture, so I suppose this needs unification. In which direction, that is something the English natives should discuss.